public/app/core/components/OptionsUI/registry.tsx

```diff
@@ -350,6 +353,32 @@ export const getAllStandardFieldConfigs = () => {
 
   const dataLinksCategory = config.featureToggles.vizActions ? 'Data links and actions' : 'Data links';
 
+  // @TODO Remove after removing the feature flag.
+  const oneClickModeOptions = [
+    { value: OneClickMode.Off, label: capitalize(OneClickMode.Off) },
+    { value: OneClickMode.Link, label: capitalize(OneClickMode.Link) },
+  ];
+
+  let oneClickDescription = 'When enabled, a single click opens the first link';
+
+  const oneClickMode: FieldConfigPropertyItem<FieldConfig, OneClickMode, SelectFieldConfigSettings<OneClickMode>> = {
+    id: 'oneClickMode',
+    path: 'oneClickMode',
+    name: 'One-click',
+    editor: standardEditorsRegistry.get('radio').editor,
+    override: standardEditorsRegistry.get('radio').editor,
+    process: (value) => value,
+    settings: {
+      options: oneClickModeOptions,
+    },
+    defaultValue: OneClickMode.Off,
+    shouldApply: () => true,
+    category: [dataLinksCategory],
+    description: oneClickDescription,
+    getItemsCount: (value) => (value ? value.length : 0),
+    showIf: () => config.featureToggles.vizActions,
+  };
```

Comment:
do we really need a new field level config for this? can't it be a datalink (item) level option, or default?

Reply:
the reason it's not on the link itself is because we're going to have oneClick Actions as well, so we need an option outside of link and action models that determines what will happen when clicking a point. while i don't think we need this specifically _per field_, that gives the most flexibility with overrides, and does not require us to put it into a panel-level option for every panel.

i can't think of a specific use case where having this be override-able is useful, but i'm not sure what we gain by moving this to a panel option and removing the flexibility.

Reply:
if you feel like overrides dont add any value here we can def move this setting up to the panel level, though the UX will be a bit odd since we cannot co-locate it near the data links and actions field config.

Reply:
Why would it have to added to every panel? And override?